packages/react/src/components/PageHeader/PageHeader.tsx

```diff
@@ -241,6 +235,176 @@ PageHeaderContent.propTypes = {
   pageActions: PropTypes.node,
 };
 
+/**
+ * ----------------
+ * PageHeaderContentPageActions
+ * ----------------
+ */
+interface PageHeaderContentPageActionsProps {
+  /**
+   * Provide child elements to be rendered inside PageHeaderContentPageActions.
+   */
+  children?: React.ReactNode;
+  /**
+   * Specify an optional className to be added to your PageHeaderContentPageActions
+   */
+  className?: string;
+  /**
+   *  The PageHeaderContent's page actions collabpsible Menu button label
+   */
+  menuButtonLabel?: string;
+  /**
+   * The PageHeaderContent's page actions
+   */
+  pageActions?: React.ReactNode;
+}
+const PageHeaderContentPageActions = React.forwardRef<
+  HTMLDivElement,
+  PageHeaderContentPageActionsProps
+>(
+  ({
+    className,
+    children,
+    menuButtonLabel = 'Actions',
+    pageActions,
+    ...other
+  }: PageHeaderContentPageActionsProps) => {
+    const prefix = usePrefix();
+    const classNames = classnames(
+      {
+        [`${prefix}--page-header__content__page-actions`]: true,
+      },
+      className
+    );
+    const containerRef = useRef<HTMLDivElement>(null);
+    const offsetRef = useRef<HTMLDivElement>(null);
+    const [menuButtonVisibility, setMenuButtonVisibility] = useState(false);
+    const [hiddenItems, setHiddenItems] = useState<any[]>([]);
```

Comment:
Can the `any` type be replaced with a fully qualified type? Asking because later on, there is the following code:

https://github.com/carbon-design-system/carbon/blob/a9f98e4d454a476acd8454c9e68f4ded8423520b/packages/react/src/components/PageHeader/PageHeader.tsx#L328-L334

Based on that code, it seems that `hiddenItems` entries should have at least `id`, `label`, and `onClick` properties.

Reply:
It's most likely going to change, but I'll define the type for now